src/styles/components/_code-pattern.scss

```diff
@@ -26,12 +26,14 @@
   pre {
     border: 0;
     background: 0;
+    white-space: pre-wrap;
   }
 
   iframe {
     border: 0;
     height: 100%;
     width: 100%;
+    min-width: 300px;
```

Comment:
It seems like you shouldn't need this?

Reply:
If for some reason iframe does not load it will break the layout with width=0